### PR TITLE
Replace personal info for publishing with Infinum's info

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Publishing info should reflect that Infinum is developer/maintainer of library. That info is shown when fetching licences with tools like: https://github.com/mikepenz/AboutLibraries 

## Changes
In all `publish.gradle` files personal info has been replaced with Infinum's info

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [x] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: sentinel-no-op/publish.gradle [hunk 1]
```diff
@@ -46,9 +46,9 @@ afterEvaluate {
                     }
                     developers {
                         developer {
-                            id = 'knobtviker'
-                            name = 'Bojan Komljenović'
-                            email = 'bojan.komljenovic@infinum.com'
+                            id = 'Infinum'
+                            name = 'Infinum Inc.'
+                            email = 'team.android@infinum.com'
```
[the remark this answers is not in the document]
Comment (KCeh) [reply sentinel-no-op/publish.gradle:51]: I think we might even remove email 
And move to 
```
organization {
                        name = 'Infinum Inc.'
                        url = 'https://infinum.com'
                    }
```

instead of developer

[the remark this answers is not in the document]
Comment (KCeh) [reply sentinel-no-op/publish.gradle]: I removed email from developer info. Since repos are public there is no barrier in reporting issue on lib.

Also, IMO we should only have _**organization**_ info. But I found that some tools that collect info for libs sometimes read only developers info and sometimes developer and organization infos are interchangeable.
This is why I added both developer and organization

[the remark this answers is not in the document]
Comment (JonatanPlesko) [reply sentinel-no-op/publish.gradle]: Makes sense, I agree 👍

**Review by JonatanPlesko — APPROVED** [review]

**Review by AsimRibo — APPROVED** [review]
Looks good. But I do think that alongside organization, notable contributors should be mentioned in developer block instead of org, per developer block semantics.

Was this agreed as part of our new template?

**Comment by KCeh** [comment]
> Looks good. But I do think that alongside organization block, notable contributors should be mentioned in developer block instead of org, per developer block semantics.
> 
> Was this agreed as part of our new template?

This is still up for debate. Developer block is not mandatory. And developer change over time so I think solution proposed here is best middle ground. Also, I saw that Google puts their name in developer block so we should be fine
